src/community/geogig/src/main/java/org/geogig/geoserver/config/RepositoryCache.java

```diff
@@ -26,6 +26,8 @@
 
     private static final Logger LOGGER = Logging.getLogger(RepositoryCache.class);
 
+    private static GeoServerGeoGigRepositoryResolver resolver = new GeoServerGeoGigRepositoryResolver();
```

Comment:
Given that separate instances of this are getting instantiated in multiple classes, should this maybe be a singleton bean instead?